docs/tutorials/advanced/classification_of_strategies.rst

```diff
@@ -69,7 +69,7 @@ range of memory_depth values, we can use the 'min_memory_depth' and
     ... }
     >>> strategies = axl.filtered_strategies(filterset)
     >>> len(strategies)
-    61
```

Comment:
I presume this means that we misclassified a number of the FSMs as finite memory?

Reply:
That's right.  Almost all FSMs were marked with memory length equal to the number of states, but actually most of them turned out to be infinite memory, since all that's needed is for the same cycle to show up in two different places.